contracts/nft/erc1155m/ERC1155MStorage.sol

```diff
@@ -0,0 +1,22 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.20;
+
+import {MintStageInfo1155} from "../../common/Structs.sol";
+
+contract ERC1155MStorage {
```

Comment:
we could but its not necessary yet since it has no functions to override, im planning another refactor of the storage contract to remove some of the duplicate functions in the 721/1155 contracts